components/eamxx/src/physics/ml_correction/eamxx_ml_correction_process_interface.cpp

```diff
@@ -99,7 +99,7 @@ void MLCorrection::initialize_impl(const RunType /* run_type */) {
 // =========================================================================================
 void MLCorrection::run_impl(const double dt) {
   // use model time to infer solar zenith angle for the ML prediction
-  auto current_ts = timestamp();
+  auto current_ts = start_of_step_ts();
```

Comment:
@AaronDonahue I did not change this, also b/c I don't know who still uses MLCorrection. That said, it seems wrong that ML uses the timestamp at the _beginning_ of the step. If you think it should switch, I can change this.

Reply:
I think this is okay for now.  We are not actively using MLCorrection so we won't break anything.  If you do have to make another commit maybe you could add a comment to this line saying exactly what you say above, that this means the CorrectiveML is using the time at the beginning of the timestep, but if not I think we will be alright.

Reply:
No other commit planned. I will integrate as is, as soon as approved, unless changes are requested.